app/controllers/health_controller.rb

```diff
@@ -5,6 +5,9 @@ def db_connection
   end
 
   def jobs_queues
+    stale_processes = GoodJob::Process.all.select(&:stale?)
```

Comment:
J’ai un peu peur que ça puisse leaker des données sensibles notamment sur la connexion à la db.
J’ai testé en prod en faisant `GoodJob::Process.first.to_json` et ça a l’air ok mais je me dis que pour nous prémunir de changements dans GoodJob plus tard on pourrait être plus restrictif : 

```suggestion
    stale_processes = GoodJob::Process.all.select(&:stale?)&.state&.fetch("hostname", "unknown-process")
```

Reply:
Oui bien vu. Je vais juste garder l'ID du process pour ne pas dépendre de la colonne `state`.